FFXIVClientStructs/FFXIV/Client/Game/UI/UIState.cs

```diff
@@ -66,7 +69,10 @@ public unsafe partial struct UIState {
     [FieldOffset(0x16398)] public GoldSaucerYell GoldSaucerYell;
     [FieldOffset(0x17AE8)] public CharaCard CharaCard;
     // 0x17CD0: ItemAction Unlocks
-
+    [FieldOffset(0x17D28)] public ushort CharaViewDrawDataFlags; // & 1 = HideHeadgear, & 2 = HideWeapons, & 4 = HideLegacyTattoo, & 0x40 = HideVisor
```

Comment:
Should this be made to an enum flag type?

Reply:
Found out what it is. :)